src/SqlBinding/SqlBinding.csproj

```diff
@@ -21,6 +21,8 @@
     <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.*" />
     <PackageReference Include="Microsoft.Data.SqlClient" Version="2.0.*" />
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.*" PrivateAssets="All" />
+    <PackageReference Include="morelinq" Version="3.3.2" />
```

Comment:
Do we have a hard requirement on this nuget package? I'm a little wary of adding 3rd party libraries to our list of dependencies if we don't have a strong reason for it.

Reply:
I use it to batch the `IEnumerable<T>`. This isn't a hard requirement and I could implement my own version of it if it's not worth the dependency